.github/workflows/Build.yml

```diff
@@ -341,6 +341,15 @@ jobs:
       - uses: actions/checkout@v3
       - uses: ./ci
 
+  klayout-osx:
+    runs-on: "macos-latest"
+    env:
+      PACKAGE: "misc/klayout"
+      OS_NAME: "osx"
```

Comment:
I assume we deliberatly don't set `PYTHON_VERSION` here like the linux job does it.

Reply:
What purpose would that serve? Wouldn't it make more sense to use the same Python version as the rest of the environment?

Reply:
I think the general idea of how `PYTHON_VERSION` is set in Build.yaml is to explicit state upfront the version of the package we want to tie the package again so that we don't end up being dependent of the builder environment (and we could add additional jobs as part of #72).

For osx we could use a different python version but still be explicit about it?

Reply:
Alright, I've added explicit packages for python-3.8 and python-3.10.